app/src/main/kotlin/io/homeassistant/companion/android/util/compose/Theme.kt

```diff
@@ -39,15 +49,94 @@ private val haDarkColors = darkColors(
  */
 @Composable
 fun HomeAssistantAppTheme(
-    content: @Composable () -> Unit
+    content: @Composable () -> Unit,
 ) {
     MaterialTheme(
-        colors = if (isSystemInDarkTheme()) haDarkColors else haLightColors
+        colors = if (isSystemInDarkTheme()) haDarkColors else haLightColors,
     ) {
         // Copied from MdcTheme:
         CompositionLocalProvider(
             LocalContentColor provides MaterialTheme.colors.onBackground,
-            content = content
+            content = content,
         )
     }
 }
+
+@Composable
```

Comment:
From this point: does it make sense to have this in the same file? You're creating a util/compose/glance package and you're not putting the theme in there.

Reply:
I made this change then went on long weekend and came back and forgot what I was doing 😂.

What is your opinion? Put everything in glance package? Or remove the glance package and move the extension somewhere else? I'm pretty sure we are going to create new extension.